Mage/Module.cs

```diff
@@ -67,28 +83,46 @@ public void SetupStaticData(Action<Exception> cb)
 		}
 
 
-		//
-		protected virtual string CommandPrefix
-		{
-			get { return null; }
-		}
+		// The module name as defined on the remote MAGE server
+		protected abstract string CommandPrefix { get; }
```

Comment:
I believe the reason why this is not enforced in the JS version is because you can create modules which are meant to be event handlers. Example: https://github.com/mage/mage-sdk-js.session/blob/master/index.js

In our case, since the current module class provides no helpers or support for defining msgStream event hooks, then its only real official purpose appears to be about making user command calls.